kbase-extension/static/kbase/js/widgets/narrative_core/publicDataSources/searchDataSource.js

```diff
@@ -22,47 +22,18 @@ define([
 ) {
     'use strict';
 
-    function objectGuidToRef (guid) {
-        var m = guid.match(/^WS:(\d+)\/(\d+)\/(\d+)$/);
-        var objectRef = m.slice(1, 4).join('/');
-        return {
-            workspaceId: parseInt(m[1]),
-            objectId: parseInt(m[2]),
-            version: parseInt(m[3]),
-            ref: objectRef,
-            dataviewId: objectRef
-        };
-    }
-
     function parseGenomeIndexV1 (item) {
-        return {
```

Comment:
searchapi2 uses a totally separate database from previous search, so index versions have been reset to 1 rather than continuing from a previous version. When we update the config.json to user searchapi2, then the previous versions are no longer relevant as those indexes are not present in the database anymore